### DISCUSSION TITLE
Update "set" commands wiki page

### DISCUSSION THREAD

**Discussion by dave-kennedy** [JetBrains/ideavim#544] [Q&A]
At least one option is missing from [the "set" commands](https://github.com/JetBrains/ideavim/wiki/%22set%22-commands) page and that is `oldundo`.

I learned about this option in [VIM-308](https://youtrack.jetbrains.com/issue/VIM-308/Undo-requires-one-more-step-if-the-cursor-is-not-at-the-position-where-it-was-after-undo). It isn't even mentioned in [CHANGES.md](https://github.com/JetBrains/ideavim/blob/master/CHANGES.md).

**Comment by AlexPl292** [comment]
Hi, thank you for the info. Actually, this is supposed to be a hidden option in case of any issues with undo, but they are fixed now. Are you using the latest version of IdeaVim?

**Reply by dave-kennedy** [reply]
I'm running the version 1.11.1. Without `nooldundo`, I get the behavior exactly as described in VIM-308. i.e., undo requires an additional step. It seems like the cursor movement is part of the undo history.
